src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -768,14 +768,6 @@ void ImageReplayer<I>::stop(Context *on_finish, bool manual, int r,
         std::swap(m_on_stop_finish, on_finish);
         m_stop_requested = true;
         m_manual_stop = manual;
-
-	Mutex::Locker timer_locker(m_threads->timer_lock);
-        if (m_delayed_preprocess_task != nullptr) {
-          canceled_task = m_threads->timer->cancel_event(
```

Comment:
@dillaman now `canceled_task` is always false and not needed any more.

Reply:
Thanks -- fixed